Docs/development/jasp-build-guide-macos.md

```diff
@@ -30,28 +31,44 @@ You can download and install Homebrew using the following command. After running
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 ```
 
+Note the instructions in mentioned under "Next Steps" after installing homebrew, where `userName` below should be changed to your username
+
+```bash
+    echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/userName/.zprofile
+    eval "$(/opt/homebrew/bin/brew shellenv)"
+```
+
 After installing Homebrew, you can start installing your requirements by running the following command in the Terminal.
 
 ```bash
 brew install conan bison flex pkg-config automake autoconf create-dmg parallel
 ```
 
+Install CMake as follows:
+
+```bash
+brew install cmake
+```
+
 > 💡 Optionally, you can install the Qt from the command line, and build JASP using that, e.g., `brew install qt`. 
 
 ### Qt Creator
+<img width="1136" alt="qtInstallComponents" src="https://user-images.githubusercontent.com/10271675/162528342-d7db3c71-9747-4e10-b193-efedb7b52191.png">
```

Comment:
This is probably more clear, but I have two issues with it. First, it is not complete, there are more items that needs to go in; second, I don't want to put a figure here because it makes it harder to maintain. I think a developer needs to be able to find an item in a list, and we should just list things that are necessary.